.github/workflows/generate_docs.yaml

```diff
@@ -0,0 +1,33 @@
+name: generate-docs
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "mkdocs.yaml"
```

Comment:
Please rename here or the file name.
https://github.com/tier4/CARET_doc/blob/main/mkdocs.yml

Reply:
Thank you for pointing out so, but I'd like to rename `mkdocs.yml` as `mkdocs.yaml`.

Reply:
f8405941a353c96e23eeef2a2e3545b705ad3891